Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackFormViewModel.swift

```diff
@@ -245,11 +242,27 @@ class SentryUserFeedbackFormViewModel: NSObject {
     lazy var logoViewWidthConstraint = sentryLogoView.widthAnchor.constraint(equalToConstant: logoWidth * config.scaleFactor)
     lazy var fullNameTextFieldHeightConstraint = fullNameTextField.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var emailTextFieldHeightConstraint = emailTextField.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
-    lazy var addScreenshotButtonHeightConstraint = addScreenshotButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var removeScreenshotButtonHeightConstraint = removeScreenshotButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var submitButtonHeightConstraint = submitButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var cancelButtonHeightConstraint = cancelButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
-    lazy var screenshotImageAspectRatioConstraint = screenshotImageView.widthAnchor.constraint(equalTo: screenshotImageView.heightAnchor)
+    lazy var screenshotImageAspectRatioConstraint = {
+        let aspectRatio: CGFloat
+        if let screenshot = self.screenshot {
+            if screenshot.size.height == 0 {
+#if !SENTRY_TEST
+                SentryLog.warning("Image had 0 height, won't be able to set a reasonable aspect ratio. Defaulting to 1:1.")
+#endif // !SENTRY_TEST
```

Comment:
`m`: Why do we want to disable this warning log in the tests? If that's spamming our test logs we could consider disabling the logs for this specific test. You could use 
```
SentryLog.withOutLogs {
...
}
```